data/json/furniture_and_terrain/furniture-appliances.json

```diff
@@ -363,6 +365,8 @@
     "move_cost_mod": 2,
     "coverage": 60,
     "required_str": 10,
+    "//": "TODO: This should be able to have this action, but it can't because it has the fireplace one",
+    "//examine_action": { "type": "appliance_convert", "furn_set": "f_null", "item": "oven" },
     "flags": [ "PLACE_ITEM", "TRANSPARENT", "EASY_DECONSTRUCT", "FIRE_CONTAINER", "CONTAINER", "BLOCKSDOOR", "MOUNTABLE" ],
```

Comment:
IF "Fire_Container" flag is the issue you should probably remove it, I think the current stand is that modern over can't actually contain fire and that this flag should go away.

Reply:
I agree in principle, but it will cause rather significant damages to players who start a new character and goes about setting a fire in a (somewhat) safe place for cooking/crafting, and suddenly finding they're burning the place down instead.
I think there's a need for some kind of warning for a fair while to let players know an ingrained behavior is no longer working as expected.

Reply:
Really it needs a different behaviour. Lighting a fire in an oven wouldn't burn the place down, it would choke the fire with smoke

Reply:
I'm happy with that. Loss of the precious first branch is a reasonably sacrifice for relearning a behavior that didn't really make sense.

Reply:
I have removed the fireplace examine action. Any more changes seem out of scope for this PR.